cs.c

```diff
@@ -674,12 +693,14 @@ static uint8_t skipdata_size(cs_struct *handle)
 		case CS_ARCH_MIPS:
 		case CS_ARCH_PPC:
 		case CS_ARCH_SPARC:
+		case CS_ARCH_RISCV:
 			// skip 4 bytes
 			return 4;
 		case CS_ARCH_SYSZ:
 			// SystemZ instruction's length can be 2, 4 or 6 bytes,
 			// so we just skip 2 bytes
 			return 2;
+		case CS_ARCH_RISCVC:
```

Comment:
few lines above, this already has CS_ARCH_RISCV skip 4 bytes. is this a mistake?

Reply:
Not at all, RISCV has a totally different mode: compress instruction(RISCVC) mode. It has no alignment restrictions and different instruction opcode. RISCVC has some implementations at LLVM upstream but I am sure it is incomplete because even in RISC-V official specification it is a changeable draft version. 
So this is a placeholder, if you mind I can clean it.